DOCSP-31359 Revise Connection Options (#941)
mongodb/docs-node: source/fundamentals/connection/connection-options.txt

   :keywords: node.js, customize

This section explains the MongoDB connection and authentication options
supported by the driver that you can set within a ``MongoClientOptions`` instance. 

For more information on setting connection options directly in a connection 
string, see :manual:`Connection Strings </reference/connection-string/>` in the 

> Server manual.

Reviewer: Nit:

```suggestion
{+mdb-server+} manual.
```

       no compression, ``1`` signifies the fastest speed, and ``9`` signifies
       the best compression. See :ref:`node-network-compression` for more information.

Additional Information
----------------------

> For more complete information on connection options that you can set within a

Reviewer: Nit:

```suggestion
To learn more about connection options that you can set within a
```
